demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/mbedtls_freertos_port.c

```diff
@@ -0,0 +1,190 @@
+/* Copyright (c) Microsoft Corporation.
+   Licensed under the MIT License. */
```

Comment:
do you need this port? you are not even building this in your cmake.

Reply:
Haven't gotten to this part in code to review as yet but there is an mbedtls port in the ESP IDF that gets pulled in without putting it into a separate cmake.  I did my crypto a little differently so I will review

Reply:
Indeed it is not needed. Removed.